specifyweb/frontend/js_src/lib/components/FormEditor/viewSpec.ts

```diff
@@ -405,9 +405,16 @@ const subViewSpec = (
       syncers.maybe(
         syncer(
           (raw: string) => {
+            const cellName = _cell.rest.node.attributes.name;
```

Comment:
I don't remember this for sure, but accessing `_cell.rest.node.attributes.name` in here might be unsafe in some cases (maybe when parsing a newly added subview, or a subview without attributes).
In the best case, parsing for each field would be kept independent of other fields as much as possible (but of course, that's not trivial in this case).

The general solution in such cases is to do as much parsing as you can here, and then finish parsing in the parent syncer object where the data from the related field is easily accessible.

I pushed a commit that refactors this like that by finishing up the sortField parsing once related table name is available. Please tell if that code makes sense or if it's too complicated and we should keep the current version

Reply:
@maxpatiiuk, 
the bug is present again with your push. I'll revert to what I had before